src/support/HttpClient.cpp

```diff
@@ -24,7 +26,8 @@ Vault::HttpClient::HttpClient(Vault::Config& config, HttpErrorCallback errorCall
 {}
 
 bool Vault::HttpClient::is_success(std::optional<HttpResponse> response) {
-  return response && response.value().statusCode.value() == 200;
+  return response &&
+    (response.value().statusCode.value() == 200 || response.value().statusCode.value() == TOKEN_TIDY_SUCCESS);
```

Comment:
If we are going to create conditional cases for the successful class of responses, it may be better to check if the response is within `200..299` rather than special casing.

Reply:
good point